R/xy-datamapping.R

```diff
@@ -7,9 +7,58 @@ XYDataMapping <- R6::R6Class(
   public = list(
     x = NULL,
     y = NULL,
-    initialize = function(x, y) {
+    plotPropertiesNames = NULL,

```

Comment:
it's going to be important do document the class members and functions to understand that the members are.

see https://github.com/Open-Systems-Pharmacology/OSPSuite-R/blob/develop/R/simulation.R#L8 for fields documentation

see https://github.com/Open-Systems-Pharmacology/OSPSuite-R/blob/develop/R/population.R#L5 for methods description